components/ai_chat/core/browser/utils.h

```diff
@@ -33,6 +35,13 @@ void GetOCRText(const SkBitmap& image, GetOCRTextCallback callback);
 #endif
 
 const std::string& GetActionTypeQuestion(mojom::ActionType action_type);
+uint32_t GetMaxPageContentLengthFromContextSize(
+    std::optional<uint32_t> context_size);
+uint32_t GetMaxPageContentLengthFromModel(const mojom::Model& model);
+uint32_t GetLongConversationWarningLengthFromContextSize(
+    std::optional<uint32_t> context_size);
+uint32_t GetLongConversationWarningLengthFromModel(const mojom::Model& model);
+bool IsValidContextSize(const std::optional<uint32_t>& context_size);
```

Comment:
Consider adding documentation for this methods here, so callers can understand their contracts.